README.md

```diff
@@ -44,11 +44,94 @@ The object has the following schema (validated [here](./lib/index.js) using [Joi
   * `done: (err: Error, key: String)` - A function that takes an error as the first parameter and the bucket key as the second parameter.
 
 **Optional**
+* `enabled: Boolean default(true)` - if `true` rate limiting will be enabled you can set it to `false` on route settings to disable rate limiting
```

Comment:
I agree "enabled" can be confusing but couldn't think of a better term.
I'll change the description as you suggested